app/manual-ledger/update-manual-ledger-with-debt-data.js

```diff
@@ -0,0 +1,51 @@
+const db = require('../data')
+const { updateQualityChecksStatus } = require('../quality-check')
+const { updatePaymentRequestCategory } = require('../payment-request')
+const { checkDebts, attachDebtInformation } = require('../debt')
+const { PASSED, AWAITING_ENRICHMENT } = require('../quality-check/statuses')
+const getManualLedgerRequestsDebt = require('../manual-ledger/get-manual-ledger-requests')
+const { AR } = require('../processing/ledger/ledgers')
+
+const updateManualLedgerWithDebtData = async (paymentRequestId) => {
+  const manualLedgerRequest = await getManualLedgerRequestsDebt(paymentRequestId)
+  if (!manualLedgerRequest) {
+    await updateQualityChecksStatus(paymentRequestId, PASSED)
+    return
+  }
+
+  const isArLedger = manualLedgerRequest.find(x => x.ledgerPaymentRequest.ledger === AR && x.ledgerPaymentRequest.value !== 0)
+  if (!isArLedger) {
+    await updateQualityChecksStatus(paymentRequestId, PASSED)
+    return
+  }
+
+  const updatedDebtData = await db.debtData.findOne({

```

Comment:
Again, create a file in `debt` to extract 1 record from `debtData` by `paymentRequestId`